keyboards/dztech/dz60rgb/info.json

```diff
@@ -0,0 +1,16 @@
+{
+  "keyboard_name": "dz60rgb",
+  "url": "",
+  "maintainer": "dz",
+  "width": 14,
+  "height": 5,
+  "layouts": {
+        "LAYOUT_60_ansi": {
```

Comment:
LAYOUT_60_ansi is a default layout for 60% boards where the bottom row is 
1.25 1.25 1.25 6.25 1.25 1.25 1.25 1.25 and the right shift is 2.75. This hot swap board is most similar to the LAYOUT_60_b_ansi that currently exists in keyboards/dz60. 

Regardless, this name, has to match the LAYOUT macro defined in dz60rgb.h and int he corresponding keymap.c files.